tests/integration/api_v3/rpc/blockchainAppsMetadata.test.js

```diff
@@ -78,7 +78,7 @@ describe('get.blockchain.apps.meta', () => {
 	});
 
 	it('returns blockchain application off-chain metadata by chainID', async () => {
-		const response = await getBlockchainAppsMetadata({ chainID: '00000001' });
+		const response = await getBlockchainAppsMetadata({ chainID: '04000000' });
```

Comment:
Please apply the same changes requested above.

Reply:
Applied.